cms/utils/compat/__init__.py

```diff
@@ -12,3 +12,4 @@
 DJANGO_2_0 = LooseVersion(DJANGO_VERSION) < LooseVersion('2.1')
```

Comment:
django 2,0 and 2.1 is not supported anymore. not even 3.0

Reply:
Same as above:

This is a backport of the original change for django 3.0. This has been done to have a direct comparison to the original change. There is another PR for django 3.1 and another for 3.2 which is a few PRs combined.